packages/core/admin/admin/src/content-manager/pages/EditView/components/DocumentActions.tsx

```diff
@@ -513,6 +533,12 @@ const PublishAction: DocumentActionComponent = ({ activeTab, id, model, collecti
         navigate({
           pathname: `../${collectionType}/${model}/${res.data.id}`,
         });
+      } else if (
+        'error' in res &&
+        isBaseQueryError(res.error) &&
```

Comment:
what is a base query error?

Reply:
A `baseQuery` is something we provide to redux-toolkit-query, it basically wraps around our axios client and handles the methods & auth etc. However, the error could be either a baseQuery error (something from the API) or a serialized error which is a generic JS mess up. So we discern between the two!